gtda/images/filtrations.py

```diff
@@ -618,3 +624,150 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class SignedDistanceFiltration(BaseEstimator, TransformerMixin):
+    """Filtrations of 2D/3D binary images based on the dilation and the erosion
+    of activated regions.
+
+    This filtration assigns to each pixel in an image a grayscale value
+    calculated as follows. For activated pixels, if the minimum Manhattan
+    distance between the pixel and any activated pixel in the image is less
+    than or equal to the parameter :param:`n_iterations`, the assigned value is
+    this distance. Otherwise, the assigned grayscale value is the sum of the
+    lengths along all axes of the image – equivalently, it is the maximum
+    Manhattan distance between any two pixels in the image. For deactivated
+    pixels, if the minimum Manhattan distance between the pixel and any
+    deactivated pixel in the image is less than or equal to the parameter
+    :param:`n_iterations`, the assigned value is the opposite of this distance.
+    Otherwise, the assigned grayscale value is the opposite of the maximum
+    Manhattan distance between any two pixels in the image.
+
+    The name of this filtration comes from the fact that it is a negatively
```

Comment:
Could you cross-check this statement in view of the above comment?

Reply:
So given the corrections above it seems to me that this should say "positively signed dilation combined with a negatively signed erosion" -- because deactivated pixels get a positive sign with the wording above. But the code seems to agree with the current statement, so do we conclude that we have to swap "deactivated with "activated" above?